moto/wafv2/responses.py

```diff
@@ -0,0 +1,51 @@
+
+from __future__ import unicode_literals
+import json
+from moto.core.utils import amzn_request_id
+
+from moto.core.responses import BaseResponse
+from .models import FakeWebACL, GLOBAL_REGION, WAFV2Backend, wafv2_backends
+from typing import List
+
+
+class WAFV2Response(BaseResponse):
+
+    @property
+    def wafv2_backend(self) -> WAFV2Backend:
+        return wafv2_backends[self.region]  # default region is "us-east-1"
+
+    @amzn_request_id
+    def create_web_acl(self):
+        """  https://docs.aws.amazon.com/waf/latest/APIReference/API_CreateWebACL.html (response syntax section) """
+
+        scope = self._get_param("Scope")
+        if scope == "CLOUDFRONT": self.region = GLOBAL_REGION
```

Comment:
We probably shouldn't override the region, as they're no way to reset this for regional calls. I've added a comment later down the line how to expose (what I expect to be) this bug.

Reply:
Thanks for the feedback! If I don't override the region, should the region be us-east-1 for all CLOUDFRONT wacls (aws only allows us-east-1)? If they are in us-east-1, then they would be in the same region as the us-east-1 REGIONAL wacls, mixing them together.